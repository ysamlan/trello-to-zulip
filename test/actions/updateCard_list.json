{
  "data": {
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One"
    },
    "card": {
      "id": "cidcidcidcidcidcidcidcid",
      "idList": "cidlcidlcidlcidlcidlcidl",
      "idShort": 1,
      "name": "Card Name"
    },
    "listAfter": {
      "id": "laidlaidlaidlaidlaidlaid",
      "name": "List After Name"
    },
    "listBefore": {
      "id": "lbidlbidlbidlbidlbidlbid",
      "name": "List Before Name"
    },
    "old": {
      "idList": "51d19203f37dc6114d0005e2"
    }
  },
  "date": "2013-11-03T20:23:46.185Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "updateCard"
}