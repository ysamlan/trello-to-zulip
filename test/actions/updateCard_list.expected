Member Creator Full Name moved card [Card Name](https://trello.com/c/cidcidcidcidcidcidcidcid) from **List Before Name** to **List After Name**